### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by sprinkledcheese**
Fixed nextdns logo in post login my.nextdns.io. Also fixed nextdns.io logo, background-image which was barely readable and fixed OUR NETWORK map.
Before: 
firefox_WG3nKyNBGX
After:
firefox_di8KxigfTh

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2289,6 +2290,22 @@ INVERT
 
 ================================
 
+nextdns.io
+
+CSS
+.index .hero .content .logo {
+    background-image: url(/static/images/logo.png);
+    background: 181a1b;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2298]: Use `${Color to be darken}` so `${white}` become `rgb(23,23,23)` -> Dark color
and use # Before it

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -2289,6 +2290,22 @@ INVERT
...
+}
+.index .key-features .feature.blocklists {
+    background-image:none !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2301]: ` background-image: none`

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -2211,6 +2211,7 @@ my.nextdns.io
 
 INVERT
 img[src="/static/media/logo-large@2x.ac61cadf.png"]
+img[src^="data:image/png;"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2214]: This a pretty global selector doesn't this mess up other elements?

Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:2214]: No, it doesn't

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:2298]: That makes the logo invisible.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:2301]: Ok, thanks.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:2298]: My bad, but it doesn't change anything from my fix.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:2298]: Okay

**Pull request merged**